<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Resume - Start Bootstrap Theme</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.15.3/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Huang Ting</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="..." /></span>
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#software">Software</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#teaching">Teaching</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Huang
                        <span class="text-primary">Ting</span>
                    </h1>
                    <div class="subheading mb-5">
                        440 Huntington Avenue · 310C West Village H · Boston, MA 02115 ·
                        <a href="mailto:name@email.com">huang.tin@northeastern.edu</a>
                    </div>
                    <p class="lead mb-5">I'm a PhD student from <a href="https://www.khoury.northeastern.edu/"> Northeastern University Khoury College of Computer Sciences</a>, advised by <a href="https://olga-vitek-lab.khoury.northeastern.edu/"> Olga Vitek</a>. I received my B.S. and M.S. degree in Software Engineering from <a href="https://en.dlut.edu.cn/"> Dalian University of Technology</a>. My research focuses on experimental design and statistical analysis of mass spectrometry-based protein quantification experiments. I'm the main developer and maintainer of R/Bioconductor pacakge <a href="https://msstats.org/msstatstmt/"> MSstatsTMT</a>.</p>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/ting-huang-b0a6b711a"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://scholar.google.com/citations?user=rUFbIE4AAAAJ&hl=en"><i class="fa fa-graduation-cap" aria-hidden="true"></i></a>
                        <a class="social-icon" href="https://github.com/huang704"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="https://twitter.com/huang704"><i class="fab fa-twitter"></i></a>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Experience-->
            <section class="resume-section" id="experience">
                <div class="resume-section-content">
                    <h2 class="mb-5">Experience</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-3">Olga Vitek's Lab, Northeastern University, USA</div>
                            <p>Develop statistical methods and open-source softwares for relative protein quantification and group comparison in mass spectrometry-based experiments with isobaric labeling (iTRAQ and TMT)</p>
                            <p>Develop simulation-based methods and open-source software for sample size estimation and optimal design of mass spectrometry-based proteomic experiments</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">05/2015-present</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Summer intern</h3>
                            <div class="subheading mb-3">Research and development team, Biognosys AG, Switzerland</div>
                            <p>Design benchmarking experiments for relative protein quantification in data-independent acquisition (DIA). </p>
                            <p>Combine precursor and fragment information for improved detection of differential abundance in data independent acquisition (DIA). </p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">05/2017-08/2017</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-3">Olga Vitek's Lab, Purdue University, USA</div>
                            <p>Develop, evaluate and implement a statistical approach for identifying multivariate but robust, interpretable and accurate panels of gene biomarkers of drug therapy response.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">08/2014-05/2015</span></div>
                    </div>
        <div class="d-flex flex-column flex-md-row justify-content-between">
            <div class="flex-grow-1">
            <h3 class="mb-0">Graduate Research Assistant</h3>
                <div class="subheading mb-3">Zengyou He's Lab, Dalian University of Technology, China</div>
                    <p>Develop a linear programming model for protein inference problem in shotgun proteomics</p>
                    <p>Develop a constrained Lasso regression approach to utilize peptide detectability for protein inference problem in shotgun proteomics</p>
            </div>
            <div class="flex-shrink-0"><span class="text-primary">09/2011-06/2014</span></div>
        </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Publications-->
            <section class="resume-section" id="publications">
                <div class="resume-section-content">
                    <h2 class="mb-5">Publications</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Maculins, E. Verschueren, T. Hinkle, P. Chang, C. Chalouni, J. Lim, A. Katakam, R. Kunz, B. Erickson, T. Huang*, et al. Proteomics of autophagy deficient macrophages reveals enhanced antimicrobial immunity via the oxidative stress response. eLife, 2021.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Huang*, M. Choi, M. Tzouros, S. Golling, N. Pandya, B. Banfai, T. Dunkley and O. Vitek. MSstatsTMT: Statistical detection of differentially abundant proteins in experiments with isobaric labeling and multiple mixtures. Molecular & Cellular Proteomics, 19:1706, 2020.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Huang*, R. Bruderer, J. Muntel, Y. Xuan, O. Vitek and L. Reiter. Combining precursor and fragment information for improved detection of differential abundance in data independent acquisition. Molecular & Cellular Proteomics, 19:421, 2020.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            M. Choi, J. Carver, C. Chiva, M. Tzouros, T. Huang*, et al. MassIVE.quant: A community resource of curated quantitative mass spectrometry-based proteomics datasets. Nature Methods, 17:981, 2020.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            J. Muntel, J. Kirkpatrick, R. Bruderer, T. Huang*, O. Vitek, A. Ori and L. Reiter. Comparison of protein quantification in a complex background by DIA and TMT workflows with fixed instrument time. Journal of Proteome Research, 18:1340, 2019.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            Z. He, T. Huang*, X. Liu, P. Zhu, B. Teng and S. Deng. Protein inference: A protein quantification perspective. Computational Biology and Chemistry, 63:21, 2016.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            Z. He, T. Huang*, C. Zhao, B. Teng. Protein Inference. Modern Proteomics–Sample Preparation, Analysis and Practical Applications, 237-42, 2016.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            B. Teng, T. Huang* and Z. He. Decoy-free protein-level false discovery rate estimation. Bioinformatics, 30:675, 2013.

                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Huang*, H. Gong, C. Yang and Z. He. ProteinLasso: a Lasso regression approach to protein inference problem in shotgun proteomics. Computational Biology and Chemistry, 43:46, 2013.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Huang* and Z. He. A linear programming model for  protein inference problem in shotgun proteomics.  Bioinformatics, 28:2956, 2012.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-circle"></i></span>
                            T. Huang*, J. Wang, W. Yu and Z. He. Protein inference: A review. Briefings in Bioinformatics, 13:586, 2012.
                        </li>
                    </ul>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Software-->
            <section class="resume-section" id="software">
                <div class="resume-section-content">
                    <h2 class="mb-5">Software</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">MSstatsTMT</h3>
                            <div class="subheading mb-3">An open-source R/Bioconductor package for protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</div>
                            <p><a href="https://msstats.org/msstatstmt/"> Website</a>, <a href="https://bioconductor.org/packages/release/bioc/html/MSstatsTMT.html"> Bioconductor</a>, <a href="https://github.com/Vitek-Lab/MSstatsTMT"> Github</a></p>
                        </div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">MSstatsSampleSize</h3>
                            <div class="subheading mb-3">An open-source R/Bioconductor package for optimal design of high-dimensional MS-based proteomics experiments</div>
                            <p><p><a href="https://msstats.org/sample-size-for-classification/"> Website</a>, <a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsSampleSize.html"> Bioconductor</a>, <a href="https://github.com/Vitek-Lab/MSstatsSampleSize"> Github</a></p>
                        </div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Teaching-->
            <section class="resume-section" id="teaching">
                <div class="resume-section-content">
                    <h2 class="mb-5">Teaching</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-university"></i></span>
                            <a href="https://computationalproteomics.khoury.northeastern.edu/"> May Institute</a>-Computation and statistics for mass spectrometry and proteomics. Two-week short course, Northeastern University, Boston MA. 2017(onsite), 2018(onsite), 2019(onsite), 2020(virtual), 2021(virtual).
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-university"></i></span>
                            Short course at the <a href="https://www.ushupo.org/Conference"> US HUPO conference</a> about "Statistical Design and Analysis of Quantitative Proteomics Experiments with TMT Labeling: Case Studies with MSstatsTMT", virtual, 2021.
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-university"></i></span>
                            Short course at the <a href="https://www.asms.org/conferences/annual-conference/annual-conference-homepage"> Annual Conference of American Society for Mass Spectrometry (ASMS)</a> about "Quantitative proteomics: case Studies", Atlanta GA, 2019.
                        </li>
                    </ul>
                </div>
            </section>
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
